### ISSUE TITLE
Format() does not support both long and abbreviated form of time zone

### ISSUE THREAD

**Issue by nathanhack** [nleeper/goment#18]
I was using`Format` to print the time zone information.  I was expecting the abbreviated form but got the long form.  I tried both "z" and "zz" with the same result.

What was done:
```
g, _ := goment.Unix(time.Now().Unix())
fmt.Println(g.Format("zz"))
fmt.Println(g.Format("z"))
```
Output expected:
"Eastern Daylight Time"
"EST"

What happened:
"Eastern Daylight Time"
"Eastern Daylight Time"

**Comment by nleeper** [comment]
Currently, z & zz only support the short timezone code. Not sure why they would print out the long form, it pulls the information directly from the Go libraries. What platform were you running on (OSX, Windows, Linux)?

**Comment by nleeper** [comment]
I've added support for the formatting token "zzzz" to support displaying the long timezone name. "z" & "zz" will continue to display the abbreviated form. This change has been released in v1.1.1. I'm going to close this issue as fixed.
